difflint/utils.py

```diff
@@ -0,0 +1,20 @@
+# TODO License goes here. (MIT)
+
+import pathlib
+
+def repo_root():
```

Comment:
The return value here should also be cached since `repo_root` is also called from lib.py.